### PR TITLE
add venue/date, cfp sections

### PR DESCRIPTION
**Pull request by strand**

### CODE REVIEW

**Comment by strand** [comment]
- [ ] remove from `feels.rb`:
```
if Sinatra::Base.development?
    …
  else
    send_file File.expand_path(…)
  end
```

- [ ] Update CFP URL to include `http://`.

**Comment by theutahkate** [comment]
Okay, this should be ready to go now. @khjrtbrg you want to check out the style changes?

**Comment by strand** [comment]
Holy merge conflict batman… I've got a few improvements, gimme five?

File: app/feels.rb [hunk 1]
```diff
@@ -5,19 +5,11 @@
 end
 
 get '/' do
-  if Sinatra::Base.development?
-    erb :index
-  else
-    send_file File.expand_path('index.html', settings.public_folder)
-  end
+  erb :index
 end
```
Comment (khjrtbrg) [inline app/feels.rb]: I'm late to the game on this, but :+1:

**Pull request merged**